core/drivers/gic.c

```diff
@@ -382,7 +382,7 @@ void gic_it_handle(struct gic_data *gd)
 
 	if (id == GIC_SPURIOUS_ID)
 		DMSG("ignoring spurious interrupt");
-	else
+	else if (id < gd->max_it)
```

Comment:
Yes, adding a trace would be better.

Having messages like "ignoring interrupt 1023" does not seems as specific as "ignoring spurious interrupt", do you think this is OK ?